Refresh the network aruba instant snmp documentation (#2918)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-aruba-instant-snmp.md

Aruba Networks is a provider of network access solutions for enterprises.

> The Centreon Monitoring Connector *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics of the Aruba Wireless Access Point equipments.

Reviewer: ```suggestion
The Centreon Monitoring Connector *Aruba Instant SNMP* relies on the SNMP protocol to query and collect status and metrics for the Aruba Wireless Access Point devices.
```

### Using a service template provided by the connector

1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).

<Tabs groupId="sync">
<TabItem value="Ap-Usage" label="Ap-Usage">

| Macro                | Description                                                                                                                                              | Default value      | Mandatory   |
|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|

> | FILTERNAME           | Filter access point name (can be a regexp)                                                                                                               |                    |             |

Reviewer: unclear

| CRITICALSTATUS       | Define the conditions to match for the status to be CRITICAL (default: '%{status} !~ /up/i'). You can use the following variables: %{status}, %{display} | %{status} !~ /up/i |             |
| WARNINGSTATUS        | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                    |                    |             |
| EXTRAOPTIONS         | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                       | --verbose          |             |

</TabItem>
<TabItem value="Ssid-Status" label="Ssid-Status">

| Macro          | Description                                                                                                                                                  | Default value          | Mandatory   |
|:---------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------|:-----------:|

> | FILTERNAME     | Filter SSID name (can be a regexp)                                                                                                                           |                        |             |

Reviewer: unclear

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_aruba_instant_snmp.pl \
	--plugin=network::aruba::instant::snmp::plugin \
	--mode=ap-usage \
	--hostname=10.0.0.1 \
	--snmp-version='2c' \
	--snmp-community='my-snmp-community'  \
	--filter-name='' \
	--warning-status='' \
	--critical-status='%{status} !~ /up/i' \
	--warning-cpu='' \
	--critical-cpu='' \

> 	--warning-clients='' \

Reviewer: thresholds?

#### Modes options

All available options for each service template are listed below:

<Tabs groupId="sync">
<TabItem value="Ap-Usage" label="Ap-Usage">

| Option                   | Description                                                                                                                                                |
|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
| --filter-counters        | Only display some counters (regexp can be used). Example: --filter-counters='^cpu$'                                                                        |

> | --filter-name            | Filter access point name (can be a regexp).                                                                                                                |

Reviewer: unclear

| --warning-status         | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                      |
| --critical-status        | Define the conditions to match for the status to be CRITICAL (default: '%{status} !~ /up/i'). You can use the following variables: %{status}, %{display}   |
| --warning-* --critical-* | Thresholds. Can be: 'total-ap', 'cpu', 'clients', 'mem-usage' (B), 'mem-usage-free' (B), 'mem-usage-prct' (%).                                             |

</TabItem>
<TabItem value="Ssid-Status" label="Ssid-Status">

| Option            | Description                                                                                                                                                     |
|:------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | --filter-name     | Filter SSID name (can be a regexp).                                                                                                                             |

Reviewer: unclear
